docs/src/pages/landing-page/Index.vue

```diff
@@ -134,13 +134,55 @@
           <div class="lp-heading lp-heading-large">
             What our community thinks of quasar
           </div>
-          <div class="text-size-20 text-lp-light text-weight-medium">
+          <div class="lp-heading lp-heading-small">
             "You've never heard of Quasar? It's the framework that made a mobile
-            app, desktop app and browser extension altogether in less than
-            12minutes."
+            app, desktop app and browser extension altogether in less than 12 minutes."
           </div>
         </div>
       </div>
+      <q-carousel
+        v-model="slide"
+        transition-prev="scale"
+        transition-next="scale"
+        swipeable
+        animated
+        navigation
+        :autoplay="true"
```

Comment:
Why is it needed to specify it as true? Isn't `autoplay` attribute ok?

Reply:
Also, if you make it `autoplay`, make it infinite too. But ask Nick if autoplay is actually needed

Reply:
the autoplay is totally an error from my part. I had some linting errors initially, I had put that there to see if it goes away, but then totally missed resetting it.